### PR TITLE
add irsa operator permissions

### PR DESCRIPTION
**Pull request by whites11**
add permissions needed to run irsa operator, including permissions needed for https://github.com/giantswarm/roadmap/issues/1546

### CODE REVIEW

File: aws-operator-role/iam-policy.json [hunk 1]
```diff
@@ -139,6 +139,65 @@
...
+        {
+            "Effect": "Allow",
+            "Action": [
+                "acm:RequestCertificate",
+                "acm:AddTagsToCertificate",
+                "acm:DescribeCertificate",
+                "acm:ListCertificates",
+                "acm:DeleteCertificate"
+            ],
+            "Resource": "*"
```
Comment (calvix) [inline aws-operator-role/iam-policy.json:200]: why do we need the certificates permission?

Comment (calvix) [reply aws-operator-role/iam-policy.json:200]: is that something new? haven't seen it on irsa policy before

Comment (whites11) [reply aws-operator-role/iam-policy.json:200]: I am working on irsa operator to be able to use a custom domain for irsa. Cloudfront needs a certificate in order to let you use a custom domain. Hence we need ACM for that

Comment (calvix) [reply aws-operator-role/iam-policy.json:200]: can you please add the new IRSA permissions also here
https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/master/capa-controller-role/irsa-operator-policy.json

File: aws-operator-role/iam-policy.json [hunk 2]
```diff
@@ -139,6 +139,65 @@
...
+                "s3:PutObjectAcl",
+                "s3:PutObject",
+                "s3:PutBucketPolicy",
+                "s3:PutBucketPublicAccessBlock",
+                "s3:PutEncryptionConfiguration",
+                "s3:PutLifecycleConfiguration",
+                "s3:PutObjectAcl"
+            ],
+            "Resource": "arn:aws:s3:::*-g8s-*"
+        },
```
Comment (njuettner) [inline aws-operator-role/iam-policy.json:190]: you might want to check line 62 and below there are already some S3 permissions

Comment (whites11) [reply aws-operator-role/iam-policy.json]: removed all duplicates, thank you

File: aws-operator-role/iam-policy.json (hunk 1 above)
Comment (whites11) [reply aws-operator-role/iam-policy.json:172]: added to CAPA role as well

**Review by calvix — APPROVED** [review]

**Pull request merged**